.gitignore

```diff
@@ -13,6 +13,12 @@
 **/target/
 Cargo.lock
 
+# js/ts
+**/node_modules
+**/coverage
+**/package-lock.json
+**/dist
+
```

Comment:
Possible moving to `sdk/.gitignore` dir.

Reply:
done https://github.com/fleek-network/ursa/pull/370/commits/784ccdcd03906e8fb266186a3e474c7d138544a4